src/sphinx_book_theme/assets/styles/sections/_article.scss

```diff
@@ -9,7 +9,8 @@
       position: relative; // needed for postion:sticky header-article
       overflow-x: unset; // needed for postion:sticky header-article
       padding: 0%;
-      max-width: 50em;
+      max-width: $content-max-width;
```

Comment:
what does it look like if we just inherit what the pydata theme does rather than defining our own?

Reply:
have reverted this back